### PR TITLE
Don't crash when image is being downloaded and integration/console tab is clicked

### PR DESCRIPTION
**Pull request by jelly**
Large images which take some time download already shown the tabs but clicking on it would result in a JavaScript error as the Container object would not yet be available.

Closes:  #1408

### CODE REVIEW

**Comment by subhoghoshX** [comment]
So it stops the expandable. Nice! It's better than expanding and going "Ooops" I guess.

Little nitpick, the presentation of it is a little weird. I guess that's for garrett. Is it possible to have the name of the container shown in the row at least?

image

**Comment by jelly** [comment]
> So it stops the expandable. Nice! It's better than expanding and going "Ooops" I guess.
> 
> I think the presentation of it is a little weird. That may be something for garrett. Is it possible to have the name of the container shown in the row at least?
> 
> image

Yes, that is unfortunate. maybe we can show it as a ListingTable without the expendable section or:

image

File: src/ImageRunModal.jsx [hunk 1]
```diff
@@ -330,7 +330,7 @@ export class ImageRunModal extends React.Component {
             tempImage.isSystem = isSystem;
             tempImage.State = { Status: _("downloading") };
             tempImage.Created = new Date();
-            tempImage.Names = [tempImage.name];
+            tempImage.Name = tempImage.name;
```
Comment (subhoghoshX) [inline src/ImageRunModal.jsx:333]: I'm not sure what this does. Changing property name/type can potentially break something somewhere, unless it's not used absolutely anywhere :-)

Comment (jelly) [reply src/ImageRunModal.jsx:333]: That is expected, the property is Name in podman's API not Names.

**Review by martinpitt — APPROVED** [review]
Thanks!

**Pull request merged**